src/components/notifier/style.css

```diff
@@ -30,4 +36,4 @@ aside.danger {
 aside.default {
 	background-color: rgb(var(--smoothly-primary-tint));
 	color: rgb(var(--smoothly-primary-contrast));
-}
+}
```

Comment:
At the end of the file we always have an empty line, though. So insert one right at the end.